Buscador de repos git(ejercicio 4 topico 1)

victorja opened the pull request:
Genera 10 links sobre el tema a buscar, avisa cuando no da resultados la busqueda y maneja errores de la api.

victorja committed:
Buscador de repos git

Genera 10 links sobre el tema a buscar, avisa cuando no da resultados la busqueda y maneja errores de la api.

pabloascencao commented on topic1ej4/script.js:
```diff
@@ -0,0 +1,60 @@
+		}
+
+	}).catch(function(error){
+		let compare = error.toString().slice(0,9); //corta los 9 primeros caracteres, serian los de TypeError, para comparar en el if
```
que pasa si el "TypeError" esta en otra posicion? me parece una validación medio fragil esta.

victorja committed:
arreglado linea 43  

Uso .indexOf para buscar por la frase "TypeError" por si no se encuentra al principio
